### PR TITLE
Order history

### PR DESCRIPTION
**Pull request by RetroSabian**

### CODE REVIEW

File: src/Components/OrderHistory/SubViews/OrderHistoryListComponent.js [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+                </View>
+                <View style= {styles.row}>
+                    <Text style= {styles.itemName}>Special instructions:</Text>
+                    <Text style= {styles.itemName}>{item.instructions}</Text>
+                </View>
+                <View style= {styles.row}>
+                    <Text style= {styles.itemName}>Date Complete:</Text>
+                    <Text style= {styles.itemName}>{item.completed_at.substring(firstIndex, eleventhIndex)}</Text>
+                </View>
+                {/* <Text style= {styles.itemName}>{item.created_at}</Text>
```
Comment (GeromeSchutte) [inline src/Components/OrderHistory/SubViews/OrderHistoryListComponent.js]: ghost code

File: src/Components/OrderHistory/SubViews/OrderHistoryListComponent.js [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+        this.setList = this.setList.bind(this);
+    }
+
+    async componentDidMount(){
+        await this.props.getOrders();
+    }  
+
+    setList(){
+        return this.props.orderHistory
+            .map(({ id, number, price, created_at, completed_at, state, instructions, line_items }) => ({
```
Comment (GeromeSchutte) [inline src/Components/OrderHistory/SubViews/OrderHistoryListComponent.js]: don't know where you use setList(), but this map literally doesn't do anything. it doesn't change the data in any way by assigning new values to fields or adding new fields. if your intent is to remove fields from the objects, you should first ask yourself whether this is necessary (what would happen if you left them there?) and if you still want to do it, there is more streamlined syntax to accomplish it.

File: src/Components/Shop/SubViews/ProductListComponent.js [hunk 3]
```diff
@@ -53,14 +53,14 @@ class ProductListComponent extends React.PureComponent {
     setFilteredList(){
         return this.props.productDetails
             .filter((item) => item.category === this.props.filterCategory)
-            .map(({ id, name, description, price, serviceProviderId, currency, varientId, category }) => ({ 
+            .map(({ id, name, description, price, serviceProviderId, currency, variantId, category }) => ({
```
Comment (GeromeSchutte) [inline src/Components/Shop/SubViews/ProductListComponent.js]: same comment as I made above

File: src/Shared/API/API.js [hunk 4]
```diff
@@ -399,6 +401,39 @@ export const spree_getSpecificTaxon = async (taxonId) => {
...
+    return await historyData.map(history => ({
+        id: history.id,
+        number: history.attributes.number,
+        price: history.attributes.display_total,
+        created_at: history.attributes.created_at,
+        completed_at: history.attributes.completed_at,
+        state: history.attributes.state,
+        intstructions: history.attributes.special_instructions,
+        line_items: history.relationships.line_items.data.map(item =>({
+            ...historyInclude.find(includedItem => includedItem.id === item.id).attributes || null
```
Comment (GeromeSchutte) [inline src/Shared/API/API.js]: what are you doing?

File: src/Components/OrderHistory/SubViews/OrderHistoryListComponent.js (hunk 1 above)
Comment (RetroSabian) [reply src/Components/OrderHistory/SubViews/OrderHistoryListComponent.js]: removed

File: src/Components/OrderHistory/SubViews/OrderHistoryListComponent.js (hunk 2 above)
Comment (RetroSabian) [reply src/Components/OrderHistory/SubViews/OrderHistoryListComponent.js]: removed

File: src/Components/Shop/SubViews/ProductListComponent.js (hunk 3 above)
Comment (RetroSabian) [reply src/Components/Shop/SubViews/ProductListComponent.js]: done

File: src/Shared/API/API.js (hunk 4 above)
Comment (RetroSabian) [reply src/Shared/API/API.js]: your code.

**Pull request merged**